lmfdb/elliptic_curves/templates/ec-index.html

```diff
@@ -210,6 +210,15 @@ <h2> Search </h2>
     <td>
     <span class="formexample"> e.g. 2 or 4-15</span>
     </td>
+    <td>
+    {{ KNOWL('ec.q.regulator',title="Regulator") }}
+    </td>
+    <td>
+    <input type='text' name='regulator' example="8.4-9.1" size=10>
```

Comment:
I think the HTML is a bit funny here, there should be a closing `</td>` before another opening one

Reply:
Fixed, thanks